queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name from animals WHERE name LIKE '%mon%';
```

Comment:
- [ ] It would be better if we change the `%mon%` to `%mon`, this will help us make the right queries